### ISSUE TITLE
ceramic preprocessing UI for all materials but with different categories

### ISSUE THREAD

**Issue by urapadmin** [arch-kiosk/arch-kiosk-office#1798]
USTP want to use the UI that they use for ceramic preprocessing for glass and metal. 
With different categories, of course. And that is the challenge. It makes no sense to duplicate the UI for every single material but perhaps I can find a way to use the same UI and make the offered categories dependent on the material. That's a bit of a challenge in FileMaker.

Also: That's the moment to switch from su_ceramics to something like material_details or so. So renaming of the table structure and some code will be necessary.

**Comment by Tfranconi** [comment]
Whichever way you can make it work--happy to explore whatever options you find, just let me know if the data are suited for import!

**Comment by Tfranconi** [comment]
Sorry, I had forgotten about this ticket

**Comment by urapadmin** [comment]
of course! This is just me cross-referencing things so that I don't forget.
